editor/components/block-list/block.js

```diff
@@ -57,8 +57,8 @@ import {
 	isMultiSelecting,
 	getBlockIndex,
 	getEditedPostAttribute,
-	getNextBlock,
-	getPreviousBlock,
+	getNextBlockUid,
```

Comment:
We need to come to a convention on camel-case for abbreviations. Elsewhere I've used e.g. `rootUID`

Related: #2511

Reply:
Right! I thought we kind of agreed on always camel casing abbreviations at some point but happy to change anyway.